### PR TITLE
 Expose full path to resource

### PR DESCRIPTION
**Pull request by Druotic**
E.g.

```
{
...
      "module": "test1.test2",
      "type": "null_resource",
      "name": "test2",
      "path": "module.test1.module.test2.null_resource.test2",
...
}
```

Exposing the `path` is very convenient for consumers and it's already in memory/computed anyway. It avoids having to try and rebuild the path based on the module, name, (data) type, etc.

### CODE REVIEW

**Comment by Druotic** [comment]
lockfile changes aren't related to my change - but I bumped them in a separate commit to capture the recent greenkeeper bumps that didn't check in the associated lockfile changes.

**Review by austinkelleher — APPROVED** [review]
Nice change

**Comment by Druotic** [comment]
Oops! Forgot docs. Added.

File: README.md [hunk 1]
```diff
@@ -121,9 +121,9 @@ The output is an object with these top-level properties:
 Each _changed resource_ has the following properties:
 
 - **`action`:** One of `"create"`, `"destroy"`, `"replace"`, `"update"`
-- **`module`:** Name of a module that contains a resource (only present when module is not `root`)
```
Comment (Druotic) [inline README.md:124]: Deleted since this is a duplicate description, and the other description is better IMO.

**Review by philidem — APPROVED** [review]

**Comment by philidem** [comment]
Published `terraform-plan-parser@1.5.0`

**Pull request merged**